doc/developer-guides/hld/hv-dev-passthrough.rst

```diff
@@ -506,7 +506,7 @@ If supported, follow these steps to enable PTM in the post-launched VM:
 
       $ acrn-dm -A -m $mem_size -s 0:0,hostbridge \
           -s 3,virtio-blk,user-vm-test.img \
-          -s 4,virtio-net,tap0 \
+          -s 4,virtio-net,tap=0 \
```

Comment:
```suggestion
          -s 4,virtio-net,tap=tap0 \
```

Reply:
As previous.
Could we use tap=LaaG to show the virtio-net format?

Reply:
change to tap=tap0